packages/apollos-data-prayer/src/prayer-requests/schema.js

```diff
@@ -3,8 +3,8 @@ import gql from 'graphql-tag';
 const prayerRequestSchema = gql`
   extend type Query {
     prayers: [PrayerRequest]
-    campusPrayers(campusId: String!): [PrayerRequest]
-    userPrayers: [PrayerRequest] @cacheControl(maxAge: 0)
```

Comment:
Does this mean that `userPrayers` will now be cached? If I add a prayer I'm pretty much going to want to see it right away.

Reply:
you will, that's a client-side cache update, this deals with the CDN. not sure why we put it here for just the `userPrayers` query